readtags.c

```diff
@@ -9,6 +9,9 @@
 /*
 *   INCLUDE FILES
 */
+#ifdef _WIN64
+#include <config.h>  /* to define _FILE_OFFSET_BITS for Windows */
```

Comment:
Can we expect config.h is always available on the platform where _WIN64 is defined?